### PR TITLE
small tweaks to allow for deployment

### PR DESCRIPTION
**Pull request by Raulp8**
-adds index.yaml file for datastore
-workaround for api_key
-adds api url for prod environment

### CODE REVIEW

File: shopsafe-backend/src/main/java/com/google/sps/data/StoreDatastoreHandler.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
 import java.io.IOException;
 import java.util.List;
 import java.util.Map;
+import java.util.ArrayList;
```
Comment (glstewart17) [inline shopsafe-backend/src/main/java/com/google/sps/data/StoreDatastoreHandler.java]: Just wondering, why was this needed?

**Review by glstewart17 — APPROVED** [review]
LGTM

File: shopsafe-backend/src/main/java/com/google/sps/data/StoreDatastoreHandler.java (hunk 1 above)
Comment (Raulp8) [reply shopsafe-backend/src/main/java/com/google/sps/data/StoreDatastoreHandler.java]: yeah, leftover code from debugging ill remove it in the next commit

Comment (glstewart17) [reply shopsafe-backend/src/main/java/com/google/sps/data/StoreDatastoreHandler.java]: No problem!

**Review by caroljli — APPROVED** [review]
LGTM
